src/js/components/Drop/Drop.js

```diff
@@ -12,6 +12,7 @@ const Drop = forwardRef(
     {
       restrictFocus,
       target: dropTarget, // avoid DOM leakage
+      trapFocus = true,
```

Comment:
Does this have the same intention of `modal` prop in Layer? if so, would we want to match the terms. If not, ignore this comment :)

https://v2.grommet.io/layer#modal

Reply:
I wouldn't say the intention is the same. Modal prevents all interaction below the layer. This just locks keyboard interaction between the modal and the window. The mouse can still interact with the page when a drop is rendered.

Reply:
Gotcha, thanks for clarifying!